README.md

```diff
@@ -11,7 +11,7 @@ Version | Status
 
 ## GeoMesa
 
-![Splash](http://geomesa.github.io/img/geomesa-overview-848x250.png)
```

Comment:
There is a lot of whitespace between the splash, build status, and gitter.im link.  @jdk2pq can you clean up the layout of this section?